examples/generate-files.sh

```
@@ -0,0 +1,522 @@
+      value: ReadOnlyMany"
+
+    # ACCESS_ROX_ACCESS_MODE contains a multiline string, so special characters need
+    # to be stripped for sed to process properly.
+    ESCAPED_ACCESS_ROX_ACCESS_MODE=$(printf '%s\n' "$ACCESS_ROX_ACCESS_MODE" \
+        | sed -e 's/[\/&]/\\&/g' -e ':a;N;$!ba;s/\n/\\n/g')
+fi
+
+
+# Affinity:
+if [[ "$CUSTOM_AFFINITY" != "" ]]; then
+    AFFINITY_INSTANCE=$(cat ${CUSTOM_AFFINITY})
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)